src/main/decompMod.F90

```diff
@@ -583,6 +756,18 @@ subroutine decompmod_clean()
     if ( allocated(clumps) )then
       deallocate(clumps)
     end if
+    if ( associated(procinfo%ggidx) )then
```

Comment:
This is another group to remove and only do the calls to figure this out in endrun.